src/par_iter/from_par_iter.rs

```diff
@@ -5,6 +5,11 @@ use std::hash::{BuildHasher, Hash};
 use std::collections::LinkedList;
 use std::collections::{BinaryHeap, VecDeque};
 
+/// `FromParallelIterator` implements the conversion to a [`ParallelIterator`].
```

Comment:
This is a conversion *from* a parallel iterator, and it should be mentioned that this is used by the `collect` method.